### PR TITLE
Added health endpoint

### PR DESCRIPTION
**Pull request by g4s8**
#392 - added health endpoint for primary storage, it returns `OK` status if check passed and `BAD_REQUEST` (it seems to be the standard error for health check errors), and JSON array response with verbose statuses for each check (only storage for now).

### CODE REVIEW

File: src/main/java/com/artipie/http/HealthSlice.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import javax.json.Json;
+import org.reactivestreams.Publisher;
+
+/**
+ * Health check slice.
+ * <p>
+ * Returns JSON with verbose status checks,
+ * response status is {@code OK} if all status passed and {@code BAD_REQUEST} if any failed.
```
Comment (olegmoz) [inline src/main/java/com/artipie/http/HealthSlice.java:46]: @g4s8 I am not sure if you are adapting to some existing health check protocol, but it does not look good to respond with `4xx` client error response. I think some `5xx` server error is more appropriate, maybe 503 Service Unavailable will make more sense?

**Review by olegmoz — CHANGES_REQUESTED** [review]
@g4s8 thanks, please see one comment

**Comment by g4s8** [comment]
@olegmoz thanks, should be fixed in https://github.com/artipie/artipie/pull/458/commits/9491048d93801030f668f47c6d78f20693f03c9d

**Review by olegmoz — APPROVED** [review]
@g4s8 thanks, I think it's good to merge

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#458` is not assigned, can't get performer

**Pull request merged**